app/models/form_profile.rb

```diff
@@ -352,25 +352,12 @@ def vets360_contact_info_hash
     return_val
   end
 
-  def pciu_disabled?
-    Flipper.enabled?(:remove_pciu, user)
-  end
-
   def initialize_contact_information
     opt = {}
     opt.merge!(vets360_contact_info_hash) if vet360_contact_info
-    if pciu_disabled?
-      # Monitor logs to validate the presence of Contact Information V2 user data
-      Rails.logger.info("VAProfile Contact Info: Address? #{opt[:address].present?},
-        Email? #{opt[:email].present?}, Phone? #{opt[:home_phone].present?}")
-    else
-      # The following pciu lines need to removed when tearing down the EVSS PCIU service.
-      opt[:email] ||= extract_pciu_data(:pciu_email)
-      if opt[:home_phone].nil?
-        opt[:home_phone] = pciu_primary_phone
-        opt[:us_phone] = pciu_us_phone
-      end
-    end
+    Rails.logger.info("VAProfile Contact Info: Address? #{opt[:address].present?},
+      Email? #{opt[:email].present?}, Phone? #{opt[:home_phone].present?}")
```

Comment:
Is this still useful to keep in the logs?

Reply:
<img width="573" alt="Screenshot 2025-06-09 at 6 02 18 PM" src="https://github.com/user-attachments/assets/7fdab13f-3816-49cd-999b-d3a712fc274b" />
I made sure to add the log cleanup task in 6b. More logs were added during the VAProfile integration testing.

Reply:
got it. sounds good 👍